schema/workflows.graphql

```diff
@@ -245,7 +266,7 @@ The status of a workflow
 union WorkflowStatus = WorkflowPendingStatus | WorkflowRunningStatus | WorkflowSucceededStatus | WorkflowFailedStatus | WorkflowErroredStatus
 
 """
-All tasks in the workflow have succeeded
+All tasks in the workflow have succeded
```

Comment:
This was correctly spelled before.

Reply:
Thanks, we'll fix the spelling so should be corrected on the next release - but won't make a release just for this :smile: